logback-classic/src/main/java/ch/qos/logback/classic/db/DBAppender.java

```diff
@@ -62,7 +63,8 @@
   static final int  CALLER_CLASS_INDEX = 12;
   static final int  CALLER_METHOD_INDEX = 13;
   static final int  CALLER_LINE_INDEX = 14;
-  static final int  EVENT_ID_INDEX  = 15;
+  static final int  ORIGIN = 15;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/tony19/logback/file/2376070365/issues/source?bid=3222618&fileBranchId=3222712#l66)